### PR TITLE
send mail when rent creation

### PR DESCRIPTION
**Pull request by meelrossi**
# Summary
Send email when a rent is created

# Screenshot
image
# Trello Card
https://trello.com/c/XaG2WXJB

### CODE REVIEW

File: app/controllers/rents_controller.rb [hunk 1]
```diff
@@ -5,7 +5,9 @@ def index
   end
 
   def create
-    if Rent.create(rent_params).valid?
+    rent = Rent.create(rent_params)
```
Comment (mishuagopian) [inline app/controllers/rents_controller.rb]: We've something wrong here. We are using `:user_id` and we should use `current_user` attribute! Thanks for the warning @francoprud !

File: .env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+MAILER_DOMAIN=training.wolox.com.ar
```
Comment (mishuagopian) [inline .env]: This file shouldn't be here :rofl:

File: app/mailers/application_mailer.rb [hunk 3]
```diff
@@ -1,4 +1,9 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'noreply@training.wolox.com.ar'
```
Comment (mishuagopian) [inline app/mailers/application_mailer.rb]: from config?

File: app/mailers/application_mailer.rb [hunk 4]
```diff
@@ -1,4 +1,9 @@
...
   layout 'mailer'
+
+  def new_rent_notification(rent)
+    @rent = rent
+    mail to: rent.user.email, subject: 'Felicitaciones! El alquiler se ha realizado con éxito.'
```
Comment (mishuagopian) [inline app/mailers/application_mailer.rb]: I18N

File: app/views/application_mailer/new_rent_notification.html.erb [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<h1>Nuevo Alquiler</h1>
```
Comment (mishuagopian) [inline app/views/application_mailer/new_rent_notification.html.erb]: We were using `.slim`, right?

File: app/views/application_mailer/new_rent_notification.html.erb [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+<p>Buenas tardes <b><%= @rent.user.first_name %> <%= @rent.user.last_name %></b></p>
+<h4>Has alquilado con exito:</h4>
```
Comment (mishuagopian) [inline app/views/application_mailer/new_rent_notification.html.erb]: I18N

File: .env (hunk 2 above)
Comment (meelrossi) [inline .env]: giphy 1

File: app/mailers/application_mailer.rb (hunk 4 above)
Comment (meelrossi) [inline app/mailers/application_mailer.rb]: esta en el otro pr 👼

File: app/views/application_mailer/new_rent_notification.html.erb (hunk 5 above)
Comment (meelrossi) [inline app/views/application_mailer/new_rent_notification.html.erb]: esta en el otro pr 👼

File: app/views/application_mailer/new_rent_notification.html.erb (hunk 6 above)
Comment (meelrossi) [inline app/views/application_mailer/new_rent_notification.html.erb]: esta en el otro pr 👼

**Pull request merged**